prisma/migrations/20220715134600_init/migration.sql

```diff
@@ -0,0 +1,73 @@
+-- CreateTable
+CREATE TABLE `PaybuttonAddress` (
+    `id` INTEGER NOT NULL AUTO_INCREMENT,
+    `address` VARCHAR(255) NOT NULL,
+    `createdAt` DATETIME(3) NOT NULL DEFAULT CURRENT_TIMESTAMP(3),
+    `updatedAt` DATETIME(3) NOT NULL,
+    `chainId` INTEGER NOT NULL,
+
+    UNIQUE INDEX `PaybuttonAddress_address_key`(`address`),
+    INDEX `PaybuttonAddress_chainId_fkey`(`chainId`),
+    PRIMARY KEY (`id`)
+) DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+
+-- CreateTable
+CREATE TABLE `Paybutton` (
+    `id` INTEGER NOT NULL AUTO_INCREMENT,
+    `name` VARCHAR(255) NOT NULL,
+    `uuid` VARCHAR(191) NOT NULL DEFAULT (uuid()),
+    `buttonData` LONGTEXT NOT NULL,
+    `providerUserId` VARCHAR(255) NULL,
+    `createdAt` DATETIME(3) NOT NULL DEFAULT CURRENT_TIMESTAMP(3),
+    `updatedAt` DATETIME(3) NOT NULL,
+
+    UNIQUE INDEX `Paybutton_name_providerUserId_unique_constraint`(`name`, `providerUserId`),
+    PRIMARY KEY (`id`)
+) DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+
+-- CreateTable
+CREATE TABLE `PaybuttonAddressOnPaybutton` (
```

Comment:
Maybe just `ButtonAddress` ?

Reply:
Would want to update the other ones below, including the field names, if we change that.

Reply:
[Prisma docs](https://www.prisma.io/docs/concepts/components/prisma-schema/relations/many-to-many-relations) use this `FoosOnBars` syntax (I actually implemented with the singular, but they use the plural). What about `AddressesOnButtons`?

Reply:
Sounds fine to me if the others agree.